index.js

```diff
@@ -72,6 +72,16 @@ app.get('/eventsStats', async (req, res, next) => {
   }
 })
 
+app.get('/alerts', async (req, res, next) => {
+  try {
+    const results = await readFile('./responses/alerts.json')
+    results.ok = results.executeAffirmations.length === 0 && results.executeSignatures.length === 0
```

Comment:
In which case `results.executeAffirmations.length` and `results.executeSignatures.length` will be zero? My understanding is that they always have at least `misbehavior`, `mostRecentTxHash` and `transactions` elements, so will not be zero.

Reply:
Fixed. Bad copy of the `eventsStats` endpoint. I'm using the existence or not of a `mostRecentTxHash` value now.